### PR TITLE
[F2F-615] - TxMA Changes

### PR DESCRIPTION
**Pull request by mcgarj**
### What changed

TxMA logging added, some payloads changed and some new TxMA events added as per [F2F-619](https://govukverify.atlassian.net/browse/F2F-615) and the various subtasks. Also listed [here](https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3485663247/TxMA+Events#CIC-CRI)

### Why did it change

It wasn't there/enough

### Issue tracking

- [F2F-615 - overarching ticket](https://govukverify.atlassian.net/browse/F2F-615)
- [F2F-616](https://govukverify.atlassian.net/browse/F2F-616)
- [F2F-617](https://govukverify.atlassian.net/browse/F2F-617)
- [F2F-618](https://govukverify.atlassian.net/browse/F2F-618)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

[F2F-619]: https://govukverify.atlassian.net/browse/F2F-619?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[F2F-616]: https://govukverify.atlassian.net/browse/F2F-616?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[F2F-617]: https://govukverify.atlassian.net/browse/F2F-617?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[F2F-618]: https://govukverify.atlassian.net/browse/F2F-618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/tests/unit/services/UserInfoRequestProcessor.test.ts [hunk 1]
```diff
@@ -334,15 +333,15 @@ describe("UserInfoRequestProcessor", () => {
 		const out: Response = await userInforequestProcessorTest.processRequest(VALID_USERINFO);
 		expect(mockCicService.getSessionById).toHaveBeenCalledTimes(1);
 		expect(mockCicService.getPersonIdentityBySessionId).toHaveBeenCalledTimes(1);
-		expect(mockCicService.sendToTXMA).toHaveBeenCalledTimes(1);
+		expect(mockCicService.sendToTXMA).toHaveBeenCalledTimes(2);
```
Comment (Caitlin-cooling) [inline src/tests/unit/services/UserInfoRequestProcessor.test.ts:336]: Could we please add a test that checks `expect(mockCicService.sendToTXMA).toHaveBeenCalledWith()` to make sure that the calls have the right details included?

Comment (Caitlin-cooling) [reply src/tests/unit/services/UserInfoRequestProcessor.test.ts:371]: Done above, didn't both for the failure scenarios

File: src/services/UserInfoRequestProcessor.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable complexity */
```
Comment (richa-misra) [inline src/services/UserInfoRequestProcessor.ts:1]: are these two eslint-disable needed?

Comment (Caitlin-cooling) [reply src/services/UserInfoRequestProcessor.ts:1]: They are needed to suppress warnings
image

File: src/services/CicService.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines-per-function */
```
[the remark this answers is not in the document]
Comment (Caitlin-cooling) [reply src/services/CicService.ts:1]: Needed to suppress warnings (although it looks like there is plenty of others in this file!)
image

**Review by richa-misra — APPROVED** [review]

**Pull request merged**